exercises/ruby-1-a/README.md

```diff
@@ -13,25 +13,26 @@ Items are added to the basket using stock-keeping unit (SKU) codes. The items in
 | Potatoes  | 10    | STAPOT |
 | Rice      | 30    | STARIC |
 | Coffee    | 14.99 | STACOF |
-| Newspaper | 2.99  | MEDNEW |
+| Newspaper |  2.99 | MEDNEW |
```

Comment:
Is this extra whitespace intentional? I think it will be stripped in the UI anyway (as double spaces get rendered as one in HTML)

Reply:
I did it to line up the decimal point. I don't feel strongly. I originally added decimals to the integers as well, but backed it out after a discussion (can't remember why now...)

Reply:
Oh, and Erik changed it back. I do like it better that way :)